src/controller/python/chip/interaction_model/__init__.py

```diff
@@ -26,7 +26,7 @@
 
 from chip.exceptions import ChipStackException
 
-__all__ = ["IMDelegate", "Status", "InteractionModelError"]
```

Comment:
Any reason to delete `IMDelegate` item from this list?

Reply:
The name 'IMDelegate' is exported by __all__ but is not defined.